### DISCUSSION TITLE
New boards on their way to amazon now. Expect to be showing stock later in the week.

### DISCUSSION THREAD

**Discussion by revk** [revk/xESP32-Faikout#88] [Announcements]
These have the 5V working so should work with more Daikin models.

![309394](https://github.com/revk/ESP32-Faikin/assets/996983/e577f842-4101-42e9-b9b1-af68301c53bc)

**Comment by ddv2005** [comment]
Does it works with units without pin  1 connected like Daikin EMURA Series?

**Reply by revk** [reply]
None of mine have pin 1 connected. Tests suggest they have a pull up on the serial data which means it works.

**Comment by trevorbattle** [comment]
How do we get one outside of the UK?

(I'm in Canada, and Amazon will not ship)

((Editing, I guess I'll use forward2me which was recommended somewhere, thanks))

**Reply by revk** [reply]
I know people tried forward2me.com

**Reply by gentlecat** [reply]
Not sure what changed, but I was able to order Faikin from Amazon.co.uk for delivery to Canada.
